### ISSUE TITLE
Make an exportable summary performance report for CCGs/practices, e.g. for CQC

### ISSUE THREAD

**Issue by HelenCEBM** [ebmdatalab/openprescribing#1434]
Could we easily implement a few-page summary of any practice or CCG's recent performance, similar to an alert but more like a report e.g. to submit to CQC? Perhaps to be printed so with few links and more charts.

E.g. Best 5 measures (inc charts for all), worst 5 measures (charts), what's getting better and worse, how your last 12-month spend compares to others etc.

**Comment by HelenCEBM** [comment]
We had a specific user request about presenting charts for CQC, and we often get requests about how to save charts (some still have to be screenshotted) - so although it's possible for users to create their own reports it can be time-consuming for them, particularly if you want to give an overview across the board beyond a specific topic, and also it doesn't include "you are improving on..". 

Printing the whole measures page is very long and lacks any emphasis of what's good and bad, whereas our alerts do not give many graphs and are not available on demand.  

We could make it really easy and quick for people to get an overview report of highlights and low-lights with a single click so that anyone can get this without having to identify and screenshot all the charts of interest. 

Of course I don't know how much use this would get but seems the kind of thing that people might not realise they need until it's there! @brianmackenna will have some thoughts.

**Comment by sebbacon** [comment]
OK thanks.

It's currently a relatively resource-intensive process to generate highlights; this will change when the matrix data store is finished.

In terms of making something that delivers on their requirements, I think we're better off rethinking measures pages *after* the matrix data store, such that they are displayed in an "interesting-only by default" format which is also printable. I also think we should allow people to customise the list of charts they see (i.e. "always show me DOACs") on this page.

In other words, implementing a report which is different from the measures dashboard seems redundant -- we should just fix the measures dashboard to be useful by default, and make sure it's printable, I think.

**Comment by brianmackenna** [comment]
Agree with all of the above. In advance of the matrix data store we should do some prep work to support any changes post it

a) "How to guides" @HelenCEBM  has done an excellent cost savings one. I think we should do a CQC one as well (probably co-authored with a CQC employee/inspector). Having these guides out in the wild will get us mountains of feedback
b) user-testing - as part of the £24k from NHSE we agreed to do a bit of user testing. I'll do this, we should plan it with above in mind

**Comment by brianmackenna** [comment]
User suggestion/request that could be considered as part of this work

> , I am currently doing a research project as part of my masters around prescribing and i am trying to ascertain the top 15 most prescribed drugs within our practice, i was hoping to use your analytics tool to do this but I cant seem to. Is there a way to do this? or would you perhaps be able to send me the data?

I used to put the Top 20 over last 12 months when I was doing my big annual visit to practices. It always prompted discussion and people were always really interested in it.

**Comment by brianmackenna** [comment]
PHE have produced reports (see Camden [examples](https://fingertips.phe.org.uk/profile-group/cardiovascular-disease-diabetes-kidney-disease/profile/cardiovascular/data#page/13/gid/1938133106/pat/46/par/E39000018/ati/152/are/E38000027)) on fingertips for some areas.
